examples/arithmetics/package.json

```diff
@@ -47,6 +47,17 @@
         "arithmetics-cli": "./bin/cli"
     },
     "main": "./out/extension.js",
+    "types": "./out/extension.d.ts",
+    "exports": {
```

Comment:
Are there any special reasons to add the exports?

Reply:
I need to access the AST types in my code gen example, and I didn't want copy it over, and lengthy imports like `langium-arithmetics-dsl/lib/language-server/generated/ast.ts` also feel odd.

But I don't insist on the export, esp. since the visible types and methods are limited to the exported ones with this `export` declaration.

Reply:
Right, I just added that for completeness as https://nodejs.org/api/packages.html#package-entry-points states that `export` might override `main`; better safe than sorry, added `types` also just for consistency.

Reply:
I see, alright.